docs/HowToUseSmartFSDumpVisualizer.md

```diff
@@ -9,8 +9,8 @@ There are three main functions in SmartFS Dump Visualizer (SDV).
 
 
 ## How to run SDV?
-To run SDV, please download and install Java to a target PC first.
-[Java Download](https://www.oracle.com/technetwork/java/javase/downloads/jdk12-downloads-5295953.html)
+To run SDV, please download and install the latest Java to a target PC first.
```

Comment:
To use RT IDE, we must install Java 8, not latest. Could you check this with Java 8?

Reply:
Updated. Thank you again!

Reply:
It still has `latest`. Could you let me know how to update?

Reply:
Now Java 8 is installed in my desktops (Windows/Linux).
I changed the explanation.

Reply:
How about saying "both java8 and latest are working, but to use RT IDE, you should install java 8"?

Reply:
Updated as recommended.